mycli/sqlexecute.py

```diff
@@ -26,8 +26,11 @@ class SQLExecute(object):
 
     version_query = '''SELECT @@VERSION'''
 
+    # System variables cannot be SELECTed (e.g. @@version_comment)
+    # this MySQL bug was fixed on `5.0.22`
+    # Bug #15684 https://bugs.mysql.com/bug.php?id=15684
```

Comment:
It's good to have this comment, but it would probably be more useful at the line with the version check rather than here.

Reply:
Hi @pasenor 

It's better to use the precise version, but we also known, there are some versions have the alpha suffix, such as `5.0.18-nt`, `5.0.20a` and so on, new complexity will be introduced here, so here we use an imprecise method to check and determine the way to retrieve the `VERSION_COMMENT`, and the alternative query is also valid for the versions like `5.0.45b` and so on, only when the version is  `>=5.1`, we use the simple `SELECT @@VERSION_COMMENT`.

Thanks and regards